src/content/app/genome-browser/components/browser-cog/BrowserCogList.tsx

```diff
@@ -73,34 +95,38 @@ export const BrowserCogList = () => {
       (action: UpdateTrackSummaryAction) => updateTrackSummary(action.payload)
     );
     return () => subscription?.unsubscribe();
-  }, [genomeBrowser]);
-
-  const cogs = Object.entries(browserCogTrackList).map(([name, pos]) => {
-    const posStyle = { top: pos + 'px' };
-
-    return (
-      <div key={name} className={styles.browserCogOuter} style={posStyle}>
-        <BrowserCog
-          cogActivated={selectedCog === name}
-          trackId={name}
-          updateSelectedCog={(trackId: string | null) =>
-            dispatch(updateSelectedCog(trackId))
-          }
-        />
-      </div>
-    );
-  });
+  }, [genomeBrowser, genomeId, objectId]);
 
-  const transformStyle = {
-    transform: 'translate(0,' + browserCogList + 'px)'
-  };
+  const cogs =
+    browserCogList &&
+    Object.entries(browserCogList).map(([name, pos]) => {
+      const posStyle = { top: pos + 'px' };
```

Comment:
We can write this as a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) but this is also alrigt